proteus/mprans/RANS3PF2D.h

```diff
@@ -10,6 +10,9 @@
 #include "ModelFactory.h"
 #include "SedClosure.h"
 #include "equivalent_polynomials.h"
+const  double DM=0.0;//1-mesh conservation and divergence, 0 - weak div(v) only
+const  double DM2=0.0;//1-point-wise mesh volume strong-residual, 0 - div(v) only
+const  double DM3=1.0;//1-point-wise divergence, 0-point-wise rate of volume change
```

Comment:
I think these could be static to avoid multiple symbol definition if this header is included in many cpp in a same library.

Reply:
These headers really shouldn't be used in more than one compilation unit, and the point here is to make the symbols local constants. They're just switches that are specific to this file (could also be macros).